paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -1012,6 +1012,11 @@ REGISTER_OP_KERNEL_WITH_CUSTOM_TYPE(conv2d_grad, MKLDNN,
                                     ops::kConvMKLDNNFP32,
                                     ops::ConvMKLDNNGradOpKernel<float, float>);
 
+REGISTER_OP_KERNEL_WITH_CUSTOM_TYPE(
+    conv2d_grad, MKLDNN, ::paddle::platform::CPUPlace, BF16,
+    ops::kConvMKLDNNFP32,
```

Comment:
kConvMKLDNNFP32 is confusing for BF16 kernels for me, will this be changed later?

Reply:
It's some kind of a convention, most of the times we have common kernel for BF16 and FP32 since they are mostly the same in oneDNN, but if it is confusing for you I can change our approach and add another enum value for BF16 after merging this PR

Reply:
I see, it is about custom kernel selection and it seems kConvMKLDNNFP32 is used for both BF16 and FP32 kernels as you said.